src/Magnum/Trade/PhongMaterialData.h

```diff
@@ -281,6 +348,15 @@ class MAGNUM_TRADE_EXPORT PhongMaterialData: public AbstractMaterialData {
          */
         UnsignedInt normalTexture() const;
 
+        /**
+         * @brief Normal texture layer
+         * @m_since_latest
+         *
+         * Available only if the material has @ref Flag::NormalTexture.
+         * @see @ref flags(), @ref AbstractImporter::texture()
+         */
+        UnsignedInt normalTextureLayer() const;
```

Comment:
The "layer" is confusing as it might also mean a layer in a layer texture. What about a "coordinate set" instead? What term does glTF / Assimp use for this?